src/components/modules/ui.ts

```diff
@@ -361,6 +361,10 @@ export default class UI extends Module<UINodes> {
       this.documentClicked(event);
     }, true);
 
+    this.readOnlyMutableListeners.on(document, 'paste', (event: ClipboardEvent) => {
+      this.handlePasteAnyBlockSelected(event);
```

Comment:
we alreay have paste hadler bound by the Paste module. Could we get rid of two similar listeners? Maybe we can leave only one.